ui/appui-react/src/appui-react/uiadmin/FrameworkUiAdmin.tsx

```diff
@@ -180,20 +173,16 @@ export class FrameworkUiAdmin extends UiAdmin {
       ? htmlElement
       : UiFramework.controls.getWrapperElement();
 
-    // istanbul ignore next
-    const hidePopup = () => {
-      IModelApp.uiAdmin.hideKeyinPalette();
-    };
+    const tools = IModelApp.tools.getToolList();
+    const keyIns = getKeyinsFromToolList(tools);
 
-    return PopupManager.showKeyinPalette(
-      this.getKeyins(),
-      el,
-      hidePopup,
-      hidePopup
-    );
+    return PopupManager.showKeyinPalette(keyIns, el);
   }
 
-  /** Hides the Key-in Palette. */
+  /**
+   * Hides the Key-in Palette.
+   * @deprecated in 4.10.x. Use {@link UiFramework.hideComponent}
```

Comment:
May want to add the same additional information on what to pass into it. Similar to `showKeyinPaletter` message.